doc/PIE.md

```diff
@@ -208,12 +208,14 @@ where zero is infinitely many, and finally the number of animation frames that f
 
 #### Animation frame lines
 
-> 	frame xpos ypos zpos xrot yrot zrot xscale yscale zscale
+> 	frame xpos zpos ypos xrot zrot yrot xscale yscale zscale
 
 Each animation line starts with a tab followed by
 
 * the serially increasing frame number
-* three (x, y, z) vectors, one for position, one for rotation, and one for scaling.
+* three 3-dimensional vectors, one for position, one for rotation, and one for scaling.
+
+NOTE: For position and rotation (but not scaling!), X and Z axes are swapped!
```

Comment:
Did you mean Y and Z are swapped?

Reply:
NOTE: For position and rotation Z denotes "up" and for scaling Y denotes "up".

Reply:
LOL, yeah I was a bit unfocused. Sorry.